test/Test.hs

```diff
@@ -1,6 +1,44 @@
+import Devtools.Brittany (getBrittanyConfig)
+import Devtools.Stylish (getStylishConfig)
 import MPrelude
+import Test.Tasty
 
 import qualified Devtools
 
 main :: IO ()
-main = Devtools.main Devtools.defaultConfig
+main = do
+  stylishConfig <- getStylishConfig empty
+
+  brittanyConfig <- getBrittanyConfig empty
+
+  let ormoluFormatter =
+        Devtools.FormatterOptions
+          { formatter = Devtools.Ormolu
+          , files = ["test/Examples/Ormolu.hs"]
+          }
+
+  let stylishFormatter =
+        Devtools.FormatterOptions
+          { formatter = Devtools.Stylish stylishConfig
+          , files = ["test/Examples/Stylish.hs"]
+          }
+
+  let brittanyFormatter =
+        Devtools.FormatterOptions
```

Comment:
Did you mean to use more than a 2 space indent here (and above)?

Reply:
They are 2 spaces from the start of `brittany`. So, I meant to use 2 spaces though I can see how it can be confused by 4. BTW shifting Devtool 2 spaces left to align with `brittany` won't parse correctly with in that `let` bind